src/style.css

```
@@ -1 +1,32 @@
-/* add styles here */
+.body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  flex-shrink: 0;
+  margin-right: 4px;
+  background: url("./images/star-active.svg") 50% no-repeat;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--0 .stars__star,
+.stars--1 .stars__star:nth-child(n + 2),
+.stars--2 .stars__star:nth-child(n + 3),
+.stars--3 .stars__star:nth-child(n + 4),
+.stars--4 .stars__star:nth-child(n + 5),
+.stars--5 .stars__star:nth-child(n + 6) {
+  width: 16px;
+  height: 16px;
+  flex-shrink: 0;
+  margin-right: 4px;
+  background: url("./images/star.svg") 50% no-repeat;
+}
```

```suggestion
.stars--0 .stars__star,
.stars--1 .stars__star:nth-child(n + 2),
.stars--2 .stars__star:nth-child(n + 3),
.stars--3 .stars__star:nth-child(n + 4),
.stars--4 .stars__star:nth-child(n + 5),
.stars--5 .stars__star:nth-child(n + 6) {
  background-image: url("./images/star.svg");
}
```
you already wrote all those styles in `.stars__star`, so here you can just change the icon